src/main/g8/scalafix/build.sbt

```diff
@@ -14,41 +18,94 @@ inThisBuild(
         url("https://example.com")
       )
     ),
-    addCompilerPlugin(scalafixSemanticdb),
-    scalacOptions ++= List(
-      "-Yrangepos",
-      "-P:semanticdb:synthetics:on"
-    )
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision
   )
 )
 
-publish / skip := true
+lazy val `$repo;format="normalize"$` = (project in file("."))
+  .aggregate(
+    rules.projectRefs ++
+      input.projectRefs ++
+      output.projectRefs ++
+      tests.projectRefs: _*
+  )
+  .settings(
+    publish / skip := true
+  )
```

Comment:
That's ugly, but without that `scalafmtAll` at the root wouldn't work... I am wondering if this couldn't trigger race conditions when reformatting `rules*` concurrently actually, but this is not specific to this PR, more to `sbt-projectmatrix`.